auth/user_mgt.go

```diff
@@ -345,10 +349,7 @@ func processClaims(p map[string]interface{}) error {
 		return nil
 	}
 
-	claims, ok := cc.(map[string]interface{})
-	if !ok {
```

Comment:
Are you sure you don't want to leave some kind of error reported here?

Reply:
This is guaranteed to be a map since we have typed functions for setting custom claims (`SetCustomClaims()` for instance).